src/mavis/pairing/pairing.py

```diff
@@ -175,10 +178,11 @@ def equivalent(event1: BreakpointPair, event2: BreakpointPair, distances=None) -
         [
             abs(Interval.dist(event1.break1, event2.break1)) > max_distance,
             abs(Interval.dist(event1.break2, event2.break2)) > max_distance,
-            event1.data[COLUMNS.event_type] != event2.data[COLUMNS.event_type],
         ]
     ):
         return False
+    if matching_event_type == True and event1.event_type != event2.event_type:
```

Comment:
The linter should yell at you for this but python likes you to use truthyness rather than == True/False so this should be

```python
if matching_event_type and event1.event_type != event2.event_type:
```

Reply:
Addressed in [0e7aad3](https://github.com/bcgsc/mavis/pull/336/commits/0e7aad362a9cbf50adc2651df62a218dd698c7ff).